### PR TITLE
Reduce the amount of columns imported on the twitter connector

### LINKED ISSUE #4674: Reduce the amount of columns imported on the twitter connector

### PR DESCRIPTION
**Pull request by Kartones**
Closes #4674

### CODE REVIEW

File: services/twitter-search/lib/twitter-search/json_to_csv_converter.rb [hunk 1]
```diff
@@ -5,39 +5,32 @@ module TwitterSearch
     class JSONToCSVConverter
 
       INDIVIDUAL_FIELDS = [
-        :id,
```
Comment (Kartones) [inline services/twitter-search/lib/twitter-search/json_to_csv_converter.rb]: This are the actual changes

File: services/datasources/lib/datasources/search/twitter.rb [hunk 2]
```diff
@@ -87,8 +87,6 @@ def initialize(config, user, redis_storage = nil, user_defined_limits={})
           }
           @redis_storage = redis_storage
 
-          @json2csv_conversor = TwitterSearch::JSONToCSVConverter.new
```
Comment (Kartones) [inline services/datasources/lib/datasources/search/twitter.rb]: This variable is no longer needed as it is sent via constructor to CSVFileDumper

**Comment by Kartones** [comment]
@ethervoid CR please

**Comment by ethervoid** [comment]
Look good to me :)

**Pull request merged**